crates/bevy_ui/src/experimental/ghost_hierarchy.rs

```diff
@@ -3,6 +3,7 @@
 use bevy_ecs::{prelude::*, system::SystemParam};
 use bevy_hierarchy::{Children, HierarchyQueryExt, Parent};
 use bevy_reflect::prelude::*;
+#[cfg(feature = "bevy_render")]
 use bevy_render::view::Visibility;
```

Comment:
Question for future PRs: Bevy `no_std` will likely introduce a lot more of these feature-gated dependencies in most crates. Stylistically, I think it makes sense to move them out of the main block of `use` statements. I personally find it harder to read, since my first thought it "_Ah the use statements below are only for `bevy_render`. Oh wait it's only this line, the others below are still always included._" Maybe not everyone has the same gut reaction but I still think there's room for a clarified style.